code/frameworks/svelte-vite/src/preset.ts

```diff
@@ -24,8 +25,23 @@ export const viteFinal: StorybookConfig['viteFinal'] = async (config, { presets
 
   plugins.push(svelteDocgen(config));
 
+  removeSvelteKitPlugin(plugins);
```

Comment:
Ideally this should have been a plugin, but it doesn't seem to have an affect. The plugin correctly removes `vite-plugin-svelte-kit` from the config but it doesn't  change any behavior